docs/admin/trial.md

```diff
@@ -49,11 +49,12 @@ Remember to create gitlab accounts for `user1` and `user2`.
     While installing you might encounter multiple dialogs asking,
     which services should be restarted. Just click **OK** to all of those.
 
-Run the following scripts.
+Run the following scripts. To setup the installation to use trial,
+the following argument is added to the script `--env trial`.
 
 ```bash
 wget https://raw.githubusercontent.com/INTO-CPS-Association/DTaaS/feature/distributed-demo/deploy/single-script-install.sh
-bash single-script-install.sh
+bash single-script-install.sh --env trial
```

Comment:
If we are using this flag for localhost installation scenario, perhaps we should call this "--localhost".

Reply:
The above scenario is for trial

The localhost scenario is `--env local`

What would the trial scenario be?